saleor/extensions/plugins/webhook/tasks.py

```diff
@@ -8,15 +8,19 @@
 from ....webhook.models import Webhook
 from . import create_webhook_headers
 
```

Comment:
Remove new line.

Reply:
This was not altered. Should the line be removed anyway?